src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/DocumentsController.java

```diff
@@ -97,4 +97,39 @@ public DocumentsController(final DocumentsService documentsService) {
             .body(new ByteArrayResource(pdfDocument));
     }
 
+    @ApiOperation("Returns a Defendant Response receipt for a given claim external id")
+    @GetMapping(
+        value = "/defendantResponseReceipt/{externalId}",
+        produces = MediaType.APPLICATION_PDF_VALUE
+    )
+    public ResponseEntity<ByteArrayResource> defendantResponseReceipt(
+        @ApiParam("Claim external id")
+        @PathVariable("externalId") @NotBlank String externalId
+    ) {
+        final byte[] pdfDocument = documentsService.generateDefendantResponseReceipt(externalId);
+
+        return ResponseEntity
+            .ok()
+            .contentLength(pdfDocument.length)
+            .body(new ByteArrayResource(pdfDocument));
+    }
+
+    @ApiOperation("Returns a Claim Issue receipt for a given claim external id")
+    @GetMapping(
+        value = "/claimIssueReceipt/{externalId}",
+        produces = MediaType.APPLICATION_PDF_VALUE
+    )
+    public ResponseEntity<ByteArrayResource> claimIssueReceipt(
+        @ApiParam("Claim external id")
+        @PathVariable("externalId") @NotBlank String externalId,
+        @ApiParam("Claim submitter email")
+        @PathVariable("submitterEmail") @NotBlank String submitterEmail
```

Comment:
@jacmurphy you declared path argument without indicating where the value should be extracted from. This basically always resolves to submitter email being null.

Other thing is that you should not expect someone to pass submitted email to the PDF generation endpoint as this is not the right place. Submitter email is evaluated at claim submission time and must not be changed afterwards.